README.rst

```diff
@@ -102,22 +102,16 @@ You'll find six directories inside the main
 Running Simulations
 -------------------
 
-With your conda environment active, the first step to running simulations
-is making the model specification files.  A model specification is a
-complete description of a vivarium model. The command to generate model
-specifications is installed with this repository and it can be run
-from any directory.::
+With your conda environment active, you can run simulations by, e.g.:
 
-  (vivarium_gates_child_iv_iron) :~$ make_specs -v
-  2020-06-18 18:18:28.311 | 0:00:00.679701 | build_model_specifications:48 - Writing model spec(s) to "/REPO_INSTALLATION_DIRECTORY/vivarium_gates_child_iv_iron/src/vivarium_gates_child_iv_iron/model_specifications"
-
-As the log message indicates, the model specifications will be written to
-the ``model_specifications`` subdirectory in this repository. You can then
-run simulations by, e.g.::
-
-   (vivarium_gates_child_iv_iron) :~$ simulate run -v /<REPO_INSTALLATION_DIRECTORY>/vivarium_gates_child_iv_iron/src/vivarium_gates_child_iv_iron/model_specifications/china.yaml
+   (vivarium_gates_child_iv_iron) :~$ simulate run -v /<REPO_INSTALLATION_DIRECTORY>/vivarium_gates_child_iv_iron/src/vivarium_gates_child_iv_iron/model_specifications/iv_iron_child.yaml
 
 The ``-v`` flag will log verbosely, so you will get log messages every time
 step. For more ways to run simulations, see the tutorials at
 https://vivarium.readthedocs.io/en/latest/tutorials/running_a_simulation/index.html
-and https://vivarium.readthedocs.io/en/latest/tutorials/exploration.html
\ No newline at end of file
+and https://vivarium.readthedocs.io/en/latest/tutorials/exploration.html
+
+You must also define the fertility input data path in your model specs yaml.
```

Comment:
Add more detailed instructions here about how to generate the birth data, where to put it, how to name those files, and how to specify that location in the configuration.